src/inspector_agent.cc

```diff
@@ -440,8 +461,10 @@ bool Agent::StartIoThread() {
 }
 
 void Agent::Stop() {
-  if (io_ != nullptr)
+  if (io_ != nullptr) {
     io_->Stop();
+    io_.reset();
```

Comment:
Stop is InspectorIo::Stop which is Node.
reset is std::unique_ptr::reset which is STL...